src/libraries/System.Web.HttpUtility/src/System/Web/HttpUtility.cs

```diff
@@ -74,7 +74,7 @@ public override string ToString()
                     }
                 }
 
-                return sb.ToString(0, sb.Length - 1);
+                return sb.Length > 0 ? sb.ToString(0, sb.Length - 1) : string.Empty;
```

Comment:
For consistency with the empty string earlier:
```suggestion
                return sb.Length > 0 ? sb.ToString(0, sb.Length - 1) : "";
```
Alternatively we could avoid appending the last `&` in the first place.